README.md

```diff
@@ -100,6 +100,49 @@ You can also use `LocalGitHubOAuthenticator` to map GitHub accounts onto local u
 
 You can use your own Github Enterprise instance by setting the `GITHUB_HOST` environment variable.
 
+### GitHub-specific features
+
```

Comment:
Perhaps something along these lines:


[GitHub scopes](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/about-scopes-for-oauth-apps/) may be used to extend the GitHub OAuthenticator. By overriding the `scope` list in the authenticator, additional features can be enabled for specific deployment needs.